apps/blog/src/posts/goodnotes-showcase/goodnotes-showcase.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: How GoodNotes uses web APIs to create a great PWA for Windows
+excerpt: Let's explore how GoodNotes uses web APIs to make their PWA great on Windows, especially on devices that support pen and touch input.
+description: In this blog post, we will explore how GoodNotes uses web APIs to make their PWA great on Windows, especially on devices that support pen and touch input.
+date: 2023-01-08
+updatedDate: 2022-01-08
+trending: true
+featured: false
+image: posts/goodnotes-showcase/person-drawing.jpg
+isPost: true
+backUrl: '/'
+author:
+  name: Justin Willis
+  title: PWABuilder SWE
+tags:
+  - post
+  - PWA
+  - showcase
+---
+
+[GoodNotes](https://www.goodnotes.com/) is a popular note-taking app that lets you create and organize handwritten notes, sketches, and PDF annotations. It has been available on iOS / ipadOS for a long time and has [recently launched a Progressive Web Application (PWA)](https://www.goodnotes.com/windows) that works on any device with a web browser, including Windows, and is in the Microsoft Store!
+
+In this blog post, we will explore how GoodNotes uses web APIs to make their PWA great on Windows, especially on devices that support pen and touch input. We will focus on two web APIs that GoodNotes utilizes: [the Ink API](https://blogs.windows.com/msedgedev/2021/08/18/enhancing-inking-on-the-web/) and the [Device Haptics API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/HapticsDevice/explainer.md).
+
+## The Ink API
+
+When Inking, aka drawing, on a device, low latency is paramount to having a good user experience. Too much latency while drawing can make writing clearly, drawing smooth lines and more difficult. Because of this, for the Web, it is paramount that your canvas code is as efficient as possible at drawing to the canvas as soon as the user presses their pen or finger to the screen.
```

Comment:
```suggestion
When Inking, or drawing, on a device, low latency is paramount to having a good user experience. Too much latency while drawing can make writing clearly, drawing smooth lines and more difficult. Because of this, for the Web, it is paramount that your canvas code is as efficient as possible at drawing to the canvas as soon as the user presses their pen or finger to the screen.
```